Tools/CISupport/ews-build/steps.py

```diff
@@ -1485,6 +1489,7 @@ def propertiesToPassToTriggers(self, patch=True, pull_request=False):
             'platform',
             'fullPlatform',
             'architecture',
+            'change_id',
```

Comment:
Don't need this. Triggered build will automatically set this property in the ConfigureBuild() step. It would already have required data (`patch_id` or `github.number`).